File: frontend/smart-fit-foods/src/app/dto/RecipeDTO.ts
import {RecipeDetailDTO} from "./RecipeDetailDTO";

export class RecipeDTO {
  food_detail?: RecipeDetailDTO[];
  name?:string;
  content?:string;
  ingredient?:string;
  steps?:string;
  defect?:string;
  advantage?:string;
  videos?:string;
  imgs?:string;

}

File: backend/src/main/java/com/example/backend/dto/RecipeDTO.java
package com.example.backend.dto;

import com.example.backend.model.recipe.Recipe;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class RecipeDTO {
    private RecipeIDetailDTO food_detail[];
    private String name;
    private String content;
    private String videos;
    private String imgs;
    private Integer rate;
}
